internal/generator/ecc/template/point.go.tmpl

```diff
@@ -1822,3 +1825,19 @@ func batchAdd{{ $TAffine }}[TP p{{ $TAffine }}, TPP pp{{ $TAffine }}, TC c{{ $TA
 		(*R)[j].Set(&rr)
 	}
 }
+
+{{ if eq .PointName "g2"}}
+func RandomOnG2() (G2Affine, error) {
```

Comment:
Should add method documentation.

Reply:
And I guess this method can be generic for both G1/G2 Affine points? Imo it could be useful.

But pinging @yelhousni in case the approach for generating random point is good and we don't introduce any security issue.